function-exercise/classExercise.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Class Exercise</title>
+        <script>
+           /* 1.) Write a JavaScript program to calculate the factorial of a number. 
+            In mathematics, the factorial of a non-negative integer n, denoted by n!, 
+            is the product of all positive integers less than or equal to n. 
+            For example, 5! = 5 x 4 x 3 x 2 x 1 = 120*/
+
+            function factorial(number) {
```

Comment:
Please use recursion and then resubmit

Reply:
Okay Sir i will try using recursion for all